doc/api/timers.md

```diff
@@ -442,6 +442,8 @@ added: v15.9.0
 -->
 
 Returns an async iterator that generates values in an interval of `delay` ms.
+If `ref` is `true`, you need to call `next()` of async iterator explicitly
+or implicitly to keep the event loop alive.
```

Comment:
I think we usually use the word "open" instead of "alive" to describe the event loop, but let's ping @nodejs/documentation for confirmation.

```suggestion
or implicitly to keep the event loop open .
```

Reply:
The document uses' active ' 🤔.